hack/update-yamlfmt.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# Copyright 2014 The Kubernetes Authors.
```

Comment:
date

Reply:
done

Reply:
still 2014?

Reply:
dang, lost the change somewhere. redoing it